src/identify/identify.ts

```diff
@@ -50,7 +50,8 @@ const defaultValues = {
   maxPushIncomingStreams: 1,
   maxPushOutgoingStreams: 1,
   maxObservedAddresses: 10,
-  maxIdentifyMessageSize: 8192
+  maxIdentifyMessageSize: 8192,
+  autodialOnConnectionOpen: true
```

Comment:
"auto dial" is quite a loaded term, plus this service doesn't dial so perhaps `runOnConnectionOpen`?